cmd/skpr/package/command.go

```diff
@@ -47,8 +48,15 @@ func NewCommand() *cobra.Command {
 	cmd.Flags().BoolVar(&command.PrintManifest, "print-manifest", command.PrintManifest, "Print the manifest to stdout.")
 	cmd.Flags().StringVar(&command.PackageDir, "dir", ".skpr/package", "The location of the package directory.")
 	cmd.Flags().StringSliceVar(&command.BuildArgs, "build-arg", []string{}, "Additional build arguments.")
-	cmd.Flags().StringVar(&command.Platform, "platform", "linux/amd64", "The platform to build for.")
 	cmd.Flags().BoolVar(&command.Debug, "debug", command.Debug, "Enable debug output.")
 
+	// See if we're using default builder.
+	userConfig, _ := user.NewClient()
+	featureFlags, _ := userConfig.LoadFeatureFlags()
```

Comment:
We've already loaded the feature flags up in the main.go

The client feature flag could get passed down into this NewCommand.